star-dash/star-dash/Info.plist

```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>UIUserInterfaceStyle</key>
```

Comment:
Please don't delete this from the info.plist lol, its to avoid text turning white during dark mode

Reply:
oops. Must have been an accident, I don't remember changing that file at all. Added it back.